acceptance/config/aio/options.rb

```diff
@@ -1,6 +1,7 @@
 {
   :type => 'aio',
   :is_puppetserver => true,
+  :puppetserver_confdir => '/etc/puppetlabs/puppetserver/conf.d',
```

Comment:
tested this against various tests that use with_puppet_running_on, and they all work fine.  the options hash is the options hash if it comes from any of the 6! places you can set options